minitrace/src/collector/global_collector.rs

```diff
@@ -127,11 +127,14 @@ pub(crate) struct GlobalCollector {
 
 impl GlobalCollector {
     fn start() -> Self {
-        std::thread::spawn(move || loop {
-            let begin_instant = std::time::Instant::now();
-            GLOBAL_COLLECTOR.lock().handle_commands();
-            std::thread::sleep(COLLECT_LOOP_INTERVAL.saturating_sub(begin_instant.elapsed()));
-        });
+        std::thread::Builder::new()
+            .name("minitrace".to_string())
```

Comment:
```suggestion
            .name("minitrace-global-collector".to_string())
```

Reply:
This is too long. Linux thread name will be truncated if it exceeds 15 characters

Reply:
First time to hear that. Learned today.